R/geom-label.R

```diff
@@ -63,7 +63,7 @@ GeomLabel <- ggproto("GeomLabel", Geom,
                         label.size = 0.25) {
     lab <- data$label
     if (parse) {
-      lab <- parse(text = as.character(lab))
+      lab <- parse_safe(text = as.character(lab))
```

Comment:
Since `parse_safe()` only has one argument, there's no longer a need to name it.